README.md

```diff
@@ -386,144 +164,5 @@ or connecting with our team in the `#ksp` channel in the
 Here are some planned features that have not yet been implemented:
 
 * Passing options to processors in Gradle.
-* Incremental processing.
 * Improved support for reading Java source code.
```

Comment:
Off topic: @ting-yuan I think we can remove these 2 items now since they are supported.

Reply:
Yes we do. Let's remove it in another PR. I'd like to remove "Migrating from old KSP releases"